src/test/scala/latis/service/hapi/DataServiceSpec.scala

```diff
@@ -5,17 +5,44 @@ import cats.effect.IO
 import cats.effect.unsafe.implicits.global
 import cats.syntax.all._
 import io.circe.syntax._
-import org.http4s._
+import org.http4s.{Status => _, _}
 import org.http4s.implicits._
 import org.scalatest.Assertion
 import org.scalatest.FlatSpec
 
+import latis.catalog.Catalog
+import latis.data.DomainData
+import latis.data.RangeData
+import latis.data.Sample
+import latis.data.SeqFunction
+import latis.dataset.MemoizedDataset
+import latis.metadata.Metadata
+import latis.model._
 import latis.service.hapi.HapiError._
-import latis.service.hapi.{Status => HStatus}
 import latis.service.hapi.HapiInterpreter.noopInterpreter
+import latis.service.hapi.{Status => HStatus}
+import latis.time.Time
+import latis.util.Identifier.IdentifierStringContext
 
 class DataServiceSpec extends FlatSpec {
 
+  /** Build a simple test DataService[IO] with a time -> int dataset using the Latis3Interpreter*/
+  private lazy val dataset = (for {
+    time <- Time.fromMetadata(Metadata("id"->"time", "type"->"string", "size"->"24", "units"->"yyyy-MM-dd", "coverage"->"2000-01-01/2000-01-05"))
```

Comment:
I'm a bit bothered by having a 10-character format string with size of 24. I assume we use `ToHapiTime` for formatting the time the way HAPI wants it. Could we (re)set the size there?  We can make this a new ticket to avoid scope creep.

Reply:
I made a ticket for this.